### ISSUE TITLE
Need a way to get seeds

### ISSUE THREAD

**Issue by heatherhaks** [InfinityRaider/AgriCraft#815]
Currently I can't seem to find a way to get seeds for certain plants, say, potato seeds. I can't put potatoes in the crafting grid to get potato seeds out. I can't simply plant potatoes with the default generated json file for potatoes. A json config for a seed item would be useful, say 'wheat_seed.json' instead of 'wheat_plant.json', and you could use that as an argument in the '*_plant.json' file instead of a seed item. Inside this '*_seed.json' file could be an optional object like the current seed item object, but perhaps also options to allow it to drop from grass. Perhaps more advanced options, to allow for more advanced features, like full auto-generation of seed graphics. Perhaps you could pick the seed type (wheat, melon, etc.) and RGB color values for a color mask applied to a default seed graphic. That way you could specify to get potato seeds you could put a potato in a crafting grid OR break grass, and you wouldn't need to make specific seed graphics.

**Comment by dragonpc75** [comment]
if you have Pam's harvestcraft installed alongside this one. you have most of the functionalities that you have described (which seeds drop from grass, as well as planting them the vanilla way (take a potato and right click the tilled ground)) and they have no seed graphics as potatoes are plantable as a potato and not a seed in vanilla.

**Comment by heatherhaks** [comment]
It was just an example, and it's bad to rely on another mod to provide functionality for your stand alone mod.

**Comment by RlonRyan** [comment]
Yes, the lack of grass drops is a known issue. The matter is complicated by the fact that forge has a way to register grass drops which puts them to some degree out of our control, to the point that you can't get the combustibility that we would like them to have.

But for now, just use a potato as the seed item for potatoes, and you have no problem. It's all largely configurable from the JSONs. And no, we do not depend on any other mods for functionality, as the expectation is instead that the JSON files handle it all.

**Comment by heatherhaks** [comment]
I know that the mod does not depend on other mods, Ryan, I was referring to dragon saying that I should rely on Pam's Harvestcraft.

**Comment by RlonRyan** [comment]
I updated the default JSON files to make potatoes obtainable. Issue #754 addresses the fact that seeds do not drop from grasses.

I tried seed texture auto-generation during the port, and didn't like the result too much. It is simpler and more flexible to just allow for any texture to be used.

The crafting recipes would have to be implemented as part of the JSON files, wherein things might get tricky.

I am closing this issue for now, as to consolidate.
